plumber/importer/mdl.py

```diff
@@ -60,25 +69,41 @@ def execute(self, context: Context) -> Set[str]:
                 editor_materials=self.editor_materials,
                 texture_format=self.texture_format,
                 texture_interpolation=self.texture_interpolation,
-                root_search=None if self.from_game_fs else (self.filepath, "models"),
+                root_search=root_search,
             )
         except OSError as err:
             self.report({"ERROR"}, f"could not open file system: {err}")
             return {"CANCELLED"}
 
         try:
-            importer.import_mdl(
-                self.filepath,
-                self.from_game_fs,
-                import_animations=self.import_animations,
-            )
+            if is_batch:
```

Comment:
This needs to check that the from_game_fs is false, since the files may contain some old previous import leftovers if it is true and this would try to import the wrong file